README.md

```diff
@@ -124,6 +124,64 @@ The bulk of newlib changes over the last several years have been in
 areas unrelated to the code used by picolibc, so keeping things in
 sync has not been difficult so far.
 
+Here's a list of some differences between picolibc and newlib:
+
+ 1. Designed for and tested on 32- and 64- bit embedded
+    systems. Newlib (and newlib-nano) are used on both embedded
+    systems and Cygwin, with the bulk of the development focused on
+    improving the Cygwin support. Because the Cygwin version needs to
+    be very careful about ABI stability, many parts of the newlib code
+    base are very difficult to change. Picolibc has no such
+    constraints, so much work has been done to improve the ABI and API
+    for use in embedded systems.
+
+ 2. Tinystdio. This new stdio implementation borrows some code from
+    avrlibc along with substantial new code. The goal of tinystdio is
+    to remain compatible with relevant ANSI-C and Posix standards
+    while using a very narrow API to the underlying system (it only
+    requires getc and putc) along with doing no internal allocation
+    and using a minimal amount of RAM (16 bytes on a 32 bit system for
+    stdin/stdout/stderr).
+
+ 3. Thread-local-storage support. Instead of creating a large data
+    structure containing all possible thread-specific data, Picolibc
```

Comment:
Consider the statement below, avoiding repeated “used”:
> Picolibc relies on the compiler’s TLS support so that only the per-thread storage actually required by the application’s API is allocated.